style.css

```diff
@@ -471,3 +479,396 @@ h2 {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .mail {
+    display: block;
+    width: 24px;
+    height: 24px;
+    background-image: url('./Images/Icon\ -\ Mail.png');
+    margin-left: 20px;
+    margin-top: 20px;
+    gap: 30px;
+  }
+
+  .close {
+    display: none;
+  }
+
+  header {
+    width: 100%;
+    height: 8px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .headline {
+    width: 100%;
+    height: 550px;
+    background-image: url("./Images/illustration-header2.png");
+    background-size: 800px 968px;
+    background-position: 100% 30%;
+  }
+
+  .brands {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 20px;
+    height: 98.96px;
+    margin-top: 150px;
+  }
+
+  .icon {
+    display: block;
+    padding: 10px;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .headi {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-left: 235px;
+    width: 566px;
+  }
+
+  .title {
+    display: block;
+    padding: 0;
+    margin-left: 220px;
+    width: 556px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: auto;
+  }
+
+  p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0;
+  }
+
+  .About-myself {
+    width: 100%;
+    height: 600px;
+  }
+
+  .Recent-work {
+    height: 52px;
+    width: 100%;
+    margin-top: -60px;
+    display: flex;
+    flex-direction: row;
+    margin-left: 153px;
+  }
+
+  .line {
+    width: 794px;
+    height: 0;
+    margin-top: 150px;
+  }
+
+  .middle {
+    display: flex;
+    flex-direction: row;
+    margin-left: 185px;
+    margin-top: 70px;
+    gap: 24px;
+  }
+
+  .about-img {
+    background-image: url(./Images/Img-Placeholder4.png);
+    background-repeat: no-repeat;
+    height: 448px;
+    width: 684px;
+  }
+
+  .rightblock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 14px;
+    height: 384px;
+    margin-top: 50px;
+  }
+
+  .Multi-Post-ul {
+    height: 40px;
+    width: 297px;
+  }
+
+  .work-container {
+    height: 448px;
+    width: 100%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    justify-items: center;
+    align-items: center;
+  }
+
+  .art-container {
+    height: 435px;
+    width: 369px;
+    margin-left: 142px;
+    background-image: url("./Images/Img-Placeholder10.png");
+  }
```

Comment:
- [ ] Please make sure to add hover interaction to the card. The hover interaction should match the figma design which means:

- **When a project card is not hovered** we can see the list of technologies, titles, description and image in the dark background:

| Figma design    | Your project design |
| --------------- | ------------------- |
| ![image](https://user-images.githubusercontent.com/74411135/236284894-6843eaf3-235a-43b9-abff-c7178357ab12.png)   | 
![image](https://user-images.githubusercontent.com/74411135/236285022-a56f59d8-e10d-4219-9de9-8ed9d9ffff6a.png)
      |

**When a card is hovered** the dark layer disappear, the title, description, list of technology disappear and the button "see project" and the background image appear:

| Figma design    | Your project design |
| --------------- | ------------------- |
| ![image](https://user-images.githubusercontent.com/74411135/236285444-e48127de-7111-40dd-93ec-0cfce05bd6cd.png)
 | ![image](https://user-images.githubusercontent.com/74411135/236285460-86dd3936-4742-4237-8606-3e667b41b633.png) |

- [ ] Kindly make sure there is some gap between all cards. Currently, there's no gap between some projects:

| Figma design    | Your project design |
| --------------- | ------------------- |
| ![image](https://user-images.githubusercontent.com/74411135/236285845-5f5ca7a3-d4fe-4b2e-a141-c808eb38af12.png) | ![image](https://user-images.githubusercontent.com/74411135/236285981-b3f2c006-985e-4e12-b7cf-c45d9209911e.png) |

This will make your project prettier and make the user know you're an excellent developer as you can precisely follow a professional web developer's design 😃 💪